### DISCUSSION TITLE
Hierarchal RSS Feeds. Can I assign a "main" or "default"list of feeds for individual users?

### DISCUSSION THREAD

**Discussion by pmisner** [FreshRSS/FreshRSS#6020] [Q&A] [answered]
I have a team who accesses my FreshRSS server to look at OSINT related feeds. Right now, my feed is public so everyone can see. In a perfect world, I would like each member of the team to login and get a default feed list, but have the ability to to create additional feeds on their own accounts. Is there a magic way to do something like this? My biggest concern is that I don't really want the site open to the public and I don't want to share a user/password with even a small team. I may be able to lock down the freshrss directory by user, but I'm worried about unforeseen consequences.

Thanks in advance for any assistance.

**Comment by Alkarex** [comment, marked as answer]
One easy approach I can think of is just to put an HTTP password at your Web server level or proxy level, with as many users as you want. This would be in addition of the Web form login of FreshRSS.

Example for Apache https://httpd.apache.org/docs/2.4/howto/auth.html
You can put the `.htaccess` file in your `./FreshRSS/p/i/` folder.

**Comment by pmisner** [comment]
Thanks. I was thinking of a direction like this.
